sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -327,44 +340,31 @@ public:
 }
 ----
 
-NOTE: `ballot_group` is templated on a `ParentGroup` because it is expected
-that it will eventually be possible to construct a ballot-group from more
-than only sub-groups.
-
 [source,c++]
 ----
 id_type get_group_id() const;
 ----
-_Returns_: An `id` representing the index of the ballot-group.
-
-NOTE: This will always be either 1 (representing the group of work-items where
-the predicate was true) or 0 (representing the group of work-items where the
-predicate was false).
+_Returns_: An `id` representing the index of the fragment within its parent.
 
 [source,c++]
 ----
 id_type get_local_id() const;
 ----
-_Returns_: An `id` representing the calling work-item's position within
-the ballot-group.
+_Returns_: An `id` representing the calling work-item's position within the
+fragment.
 
 [source,c++]
 ----
 range_type get_group_range() const;
 ----
-_Returns_: A `range` representing the number of ballot-groups.
-
-NOTE: This will always return a `range` of 2, as there will always be two groups;
-one representing the group of work-items where the predicate was true and
-another representing the group of work-items where the predicate was false.
-It is possible for one of these groups to contain no work-items, if the
-predicate has the same value on all work-items.
+_Returns_: A `range` representing the number of fragments created when the
+parent group was partitioned.
```

Comment:
`get_opportunistic_group` does not explicitly define what a parent group is and it doesn't use "partitioning" term either, so I assume that definitions as such could be slightly confusing when it comes to fragments produced that way.

Reply:
I've tried to clarify this in a4b38be5d2.  I think we may need to revisit this wording eventually -- I would not be surprised if people asked us for a way to create a `fragment` directly from something like a mask, which would behave more like `get_opportunistic_group()` than a partionining.